### PR TITLE
fix:(installer): package web player

### PR DESCRIPTION
**Pull request by thenextman**

### CODE REVIEW

File: package/Linux/gateway/template/rules [hunk 1]
```diff
@@ -12,5 +12,6 @@ override_dh_install:
 	dh_install
 	mkdir -p $$(pwd)/debian/devolutions-gateway/usr/share/devolutions-gateway/webapp
 	cp -r {{ dgateway_webclient }} $$(pwd)/debian/devolutions-gateway/usr/share/devolutions-gateway/webapp
+	cp -r {{ root_path }}/webapp/player $$(pwd)/debian/devolutions-gateway/usr/share/devolutions-gateway/webapp
```
Comment (pauldumais) [inline package/Linux/gateway/template/rules:15]: I think you are missing /player at the end of this line?

Comment (thenextman) [reply package/Linux/gateway/template/rules:15]: Screenshot is the structure of the .deb produced in CI/Packaging

Comment (pauldumais) [reply package/Linux/gateway/template/rules:15]: Oh yes sorry about that you are correct.

**Review by pauldumais — APPROVED** [review]
LGTM

**Pull request merged**